## Configuration file used by the gripper mover

group_name:   motor_name

joint_name:   motor_joint

joint_limits: [-6.28, 6.28]  ## rad

vel_limit:    2.0            ## rad/s -> adviced 0.825

acc_limit:    4.0            ## rad/s^2
 
ctrl_rate:    500            ## Hz

inst_target:  false          ## set to false if target is reached by 
                             ## non instantaneous trajectory